TestProject/gameoflife.php

```diff
@@ -2,36 +2,59 @@
 use cellularAutomat\Board;
 use options\Getopt;
 
+use input\Base;
+use input\Glider;
+use input\Random;
+
+$width = 10;
+$height = 10;
+$maxSteps = 20;
 
 /**
  * Function for Autoloader.
  * @param $className
  */
 function myAutoload($className) {
-    $classPath = str_replace("\\", "/", $className);
-    require_once(sprintf("%s/%s.php", __DIR__, $classPath));
+    $classPath = sprintf("%s/%s.php", __DIR__, str_replace("\\", "/", $className));
+    if (is_readable($classPath)) require_once($classPath);
 }
 
 /**
- * Execution of the autoloader
+ * Function for the autoloader.
  */
 spl_autoload_register("myAutoload");
 
+$options = new Getopt(
+    [
+        ['i', "input", Getopt::REQUIRED_ARGUMENT, " Which input should be used to fill the board"],
+        ['w', "width", Getopt::REQUIRED_ARGUMENT, "Sets the width of the board"],
+        ['e', "height", Getopt::REQUIRED_ARGUMENT, "Sets the height of the board"],
+        ['s', "maxSteps", Getopt::REQUIRED_ARGUMENT, "Sets the number of the generations"],
+        ['v', "version", Getopt::NO_ARGUMENT, "Shows the version"],
+        ['h', "help", Getopt::NO_ARGUMENT, "Shows a help/guide menu"]
+    ]);
+
+
+$files = glob("input/*.php");
```

Comment:
Add a small comment above that explain what the following lines do.

Reply:
Now that i reread the documentation i find it quite redundant. A good PHP-programmer knows what the glob function does. What you should describe is what the code tries to accomplish, also not only the glob line but also the foreach after it. Please try to find a better sentence.